src/core/namespace.c

```diff
@@ -319,8 +327,11 @@ static int apply_mount(
         r = mount(what, m->path, NULL, MS_BIND|MS_REC, NULL);
         if (r >= 0)
                 log_debug("Successfully mounted %s to %s", what, m->path);
-        else if (m->ignore && errno == ENOENT)
-                return 0;
+        else {
+                if (m->ignore && errno == ENOENT)
+                        return 0;
+                log_error_errno(errno, "Failed mounting %s to %s: %s", what, m->path, strerror(errno));
```

Comment:
I'd like `log_open() ... log_close()` instead of [the cryptic message](https://github.com/systemd/systemd/pull/3685#issuecomment-232239092):
```
haveged.service: Failed at step NAMESPACE spawning /usr/sbin/haveged: No such file or directory
```
But https://github.com/systemd/systemd/pull/3685#discussion_r70574750
> Hence, please return a useful error here and let the caller log about it here too.

So, I don't know